### PR TITLE
[TD-12849]<docs>(taosAdapter): document refine

### PR DESCRIPTION
**Pull request by sangshuduo**

### CODE REVIEW

**Review by huskar-t — APPROVED** [review]
LGTM

File: BUILD-CN.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+taosAdapter 是由 Go 语言开发的。关于 golang 开发环境的设置，请参考 golang 的[官方文档](https://go.dev/learn/)。
+
+请使用1.14以上版本的 golang。对于中国的用户，我们建议使用代理服务来加速软件包的下载。
+```
+go env -w GO111MODULE=on
+go env -w GOPROXY=https://goproxy.cn,direct
+```
+
+## 作为 TDengine 的一个组件构建 taosAdapter
+taosAdapter 的源代码是作为一个独立的代码库托管的，也通过子模块的方式存在于 TDengine 中。您可以下载 TDengine 的源代码并同时构建它们。步骤如下：
```
Comment (zitsen) [inline BUILD-CN.md:17]: Better to use consistant section style, place empty line between header and content.

File: BUILD-CN.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+```
```
Comment (zitsen) [inline BUILD-CN.md:18]: add language property for code block , eg. shell or bash or sh

File: BUILD-CN.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+cd debug
+cmake ..
+make
+sudo make install
+```
+
+一旦 make install 完成，taosAdapter 和它的 systemd 服务文件就会被安装到有 TDengine 服务端软件的系统中。您可以使用 sudo systemctl start taosd 和 sudo systemclt stop taosd 来启动它们。
+
+##  单独构建 taosAdapter
+如果您已经部署了 TDengine 服务器 v2.3.0.0 或以上的版本，taosAdapter 也可以作为一个独立的应用程序被构建。
```
Comment (zitsen) [inline BUILD-CN.md:32]: here.

File: BUILD-CN.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## 安装 TDengine 服务器或客户端安装包
+请从官方网站下载 TDengine 服务器或客户端安装包。
```
Comment (zitsen) [inline BUILD-CN.md:35]: here

File: BUILD-CN.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## 构建 taosAdapter
+```
+git clone https://github.com/taosdata/taosadapter
+cd taosadapter
+go build
+```
+
+然后您应该在工作目录中找到 taosAdapter 的二进制可执行文件。您需要将 systemd 服务配置文件 taosadapter.service 复制到 /etc/systemd/system 目录，并将可执行的 taosAdapter 二进制文件复制到 Linux 的 $PATH 环境变量可以找到的路径下。
```
Comment (zitsen) [inline BUILD-CN.md:44]: wrap `$PATH`-like things as inline-code.

File: README-CN.md [hunk 6]
```diff
@@ -183,7 +139,7 @@ object OpenTsdbWriter "opentsdb" {
 
 ### TCollector
 
-Tcollector 是一个客户端进程，它从本地收集器中收集数据并将数据推送到 OpenTSDB。你在你的所有主机上运行它，它完成将每个主机的数据发送到 TSD （OpenTSDB 后台服务进程）的工作。
+Tcollector 是一个客户端进程，它从本地收集器中收集数据并将数据推送到 OpenTSDB。您在您的所有主机上运行它，它完成将每个主机的数据发送到 TSD （OpenTSDB 后台服务进程）的工作。
```
Comment (zitsen) [inline README-CN.md:142]: TCollector ?

**Review by zitsen — COMMENTED** [review]
Commented.

**Review by zitsen — COMMENTED** [review]
Commented.

File: BUILD-CN.md (hunk 1 above)
Comment (sangshuduo) [reply BUILD-CN.md:17]: OK

**Pull request merged**